crates/ark/src/lsp/completions/sources/unique/custom.rs

```diff
@@ -302,18 +302,34 @@ mod tests {
             let (text, point) = point_from_cursor("Sys.getenv(@)");
             assert_has_ark_test_envvar_completion(text.as_str(), point);
 
+            // Inside the parentheses, multiline
```

Comment:
Here I just interleaved the multiline scenarios inside the existing test. This is the `Sys.getenv()` custom completion case.